src/components/Profile/components/PersonalInfoList/UpdateUserPrivacyDialog/index.jsx

```diff
@@ -0,0 +1,80 @@
+import { FormControl, IconButton, Input, InputLabel } from '@mui/material';
+import GordonSnackbar from 'components/Snackbar';
+import { useState, useEffect } from 'react';
+import userService from 'services/user';
+import SearchField from 'views/PeopleSearch/components/SearchFieldList/components/SearchField';
+
+const UpdateUserPrivacy = (username, field) => {
+  const [snackbar, setSnackbar] = useState({ message: '', severity: null, open: false });
+  const [group, setGroup] = useState('');
+  const [groupList, setGroupList] = useState([]);
+  let visibleTo = '';
```

Comment:
```suggestion
  let visibleTo = '';
```

please modify this to use states. Holding values within variables is not very reliable.